internal/constants/constants.go

```diff
@@ -27,4 +27,6 @@ const (
 	SubmarinerName          = "submariner"
 	SubmarinerNamespace     = "submariner-operator"
 	TrueLabel               = "true"
+	SubmarinerName          = "submariner"
+	SubmMissingMessage      = "Submariner is not installed"
```

Comment:
This is only used once, I don’t think it needs to be a constant.

Reply:
Not exactly, it will be used in `show` and other `diagnose` commands too.